src/components/switch/switch.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import styles from './switch-styles.scss';
+
+class Switch extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { selectedOption: props.selectedOption };
+  }
+
+  handleOnClick(option) {
+    const { onClick } = this.props;
+    this.setState({ selectedOption: option.value });
+    onClick();
+  }
+
+  render() {
+    const { options, theme } = this.props;
+    const { selectedOption } = this.state;
+    const hasOptions = options && options.length > 1;
+    const checkedStyle = cx(styles.checkedOption, theme.checkedOption);
+
+    return (
+      <div className={cx(styles.wrapper, theme.wrapper)}>
```

Comment:
I think worth it to have a different markup to be more semantic. So I'd probably go for a `<ul>` list with inputs type radio and labels inside.